File: 프로그래머스/lv0/120886. A로 B 만들기/A로 B 만들기.py
def solution(before, after):
    dic={}
    answer = 0
    for e in before :
        if e not in dic :
            dic[e] = 1
        else :
            dic[e] += 1
    for e in after :
        if e in dic :
            dic[e] -= 1
    for val in dic.values() :
        if val != 0:
            answer += 1
    answer = 1 if answer == 0 else 0
    return answer 

File: 프로그래머스/lv0/120886. A로 B 만들기/A로 B 만들기.java
import java.util.*;
class Solution {
    public int solution(String before, String after) {
        int answer = 0;
        char[] b = before.toCharArray();
        char[] a = after.toCharArray();
        Arrays.sort(b);
        Arrays.sort(a);
        
        if(String.valueOf(b).equals(String.valueOf(a))) answer = 1;
        return answer;
    }
}